examples/tutorials/manage-ntp.markdown

```diff
@@ -0,0 +1,741 @@
+---
+layout: default
+title: Manage Network Time Protocol
+published: true
+sorting: 3
+tags: [getting started, tutorial]
+---
+
+In this tutorial we will write a simple policy to ensure that the latest version of the NTP service is installed on your system. Once the NTP software is installed, we will extend the policy to manage the service state as well as the software configuration.
+
+Note: For simplicity, in this tutorial we will work directly on top of the Masterfiles Policy Framework (MPF) in `/var/cfengine/masterfiles` (*masterfiles*) and we will not use version control.
+
+## Ensuring the NTP package is installed
+
+
+```cf3
+bundle agent ntp
+{
+   vars:
+       "ntp_package_name" string => "ntp";
+
+   packages:
+       "$(ntp_package_name)"   -> { "StandardsDoc 3.2.1" } 
+       policy          => "present",
+       package_module  => yum,
```

Comment:
Well, I did that because the next section in the original tutorial was to adjust the promise to cover multiple platforms.

I could yank that entire section and rely on the MPF defaults. I don't have a strong opinion.

What do you think?

Reply:
@olehermanse @craigcomstock

Reply:
Simplify and rely on MPF

Reply:
OK, redacted